services/QuillLMS/app/controllers/teachers/progress_reports/activity_sessions_controller.rb

```diff
@@ -151,15 +152,32 @@ def index
     end
   end
 
+  private def calculate_timespent(seconds)
```

Comment:
Small quibble but I would call this method something like "timespent_to_text" since you aren't actually calculating anything here.

Reply:
updated!